cli/src/main/scala/scalafix/cli/Cli.scala

```diff
@@ -79,6 +90,16 @@ case class ScalafixOptions(
     case Some(x) => x
   }
 
+  lazy val resolvedMirror: Option[ScalafixMirror] = (classpath, sourcepath) match {
+    case (Some(cp), Some(sp)) =>
+      Some(ScalafixMirror.fromMirror(scala.meta.Mirror(cp, sp)))
+    case (None, None) => None
+    case _ =>
+      // FIXME: improve during review.
+      throw new Exception(
```

Comment:
We can move this into the `parse` method, returning `Left(errorMessage)` . See https://github.com/scalacenter/scalafix/blob/9cbc036d0fa8fd8f0f388d9d3ea8909e8463bcb2/cli/src/main/scala/scalafix/cli/Cli.scala#L179